corehq/apps/export/forms.py

```diff
@@ -1002,9 +999,6 @@ def get_form_filter(self, mobile_user_and_group_slugs, can_access_all_locations,
             self.cleaned_data['date_range'],
         )
 
-    def _get_selected_user_types(self, mobile_user_and_group_slugs):
-        return self._get_mapped_user_types(self._get_selected_es_user_types(mobile_user_and_group_slugs))
```

Comment:
:+1: There's a lot of indirection in this code - sometimes it's nice as an abstraction, but other times it just makes everything one step more complex